eng/config/PublishData.json

```diff
@@ -54,16 +54,7 @@
       "vsBranch": "rel/d16.4",
       "vsMajorVersion": 16
     },
-    "release/dev16.5-preview2-vs-deps": {
-      "nugetKind": ["Shipping", "NonShipping"],
-      "version": "3.5.*",
-      "nuget": [ "https://dotnet.myget.org/F/roslyn/api/v2/package" ],
-      "vsix": [ "https://dotnet.myget.org/F/roslyn/vsix/upload" ],
-      "channels": [ "dev16.5", "dev16.5p2" ],
-      "vsBranch": "rel/d16.5",
-      "vsMajorVersion": 16
-    },
-    "master-vs-deps": {
+    "release/dev16.5-vs-deps": {
```

Comment:
this should have the "dev16.5p3" channel instead

Reply:
https://github.com/dotnet/roslyn/pull/41498